omod/src/main/web/dashboardwidgets/obsgraph/obsgraph.controller.js

```diff
@@ -209,6 +209,10 @@ export default class ObsGraphController {
                 conceptObject.values = {};
                 for (let k = 0; k < obsArray.length; k++) {
                   let obs = obsArray[k];
+                  //Skip obs if encounter type does not match (only when encounter type specified in config)
+                  if(angular.isDefined(self.encounterType) && angular.isDefined(obs) && obs.encounter.encounterType.uuid !== self.encounterType){
```

Comment:
Thanks @rrameshbtech, I would think an implementation may want to configure more than `encounterType` which you may have to support her as well. 

cc: @mks-d, @mogoodrich

Reply:
Added support for more than one encounter types.